maxwell_slice/sphere_scat.py

```diff
@@ -16,6 +17,9 @@ def sphere_scat():
         # Here is where we run the fortran program using a bash script
         s = ShellScript(f'''
         #!/bin/bash
```

Comment:
We need to make sure that we have the proper working directory for this script, in order to find the fortran program.